### PR TITLE
[AMORO-1744]Change project name to `Amoro` in the GitHub templates

### PR DESCRIPTION
**Pull request by zhoujinsong**
## Why are the changes needed?

Change the project name from `Arctic` to `Amoro` in the GitHub template files as #1744 said.

## Brief change log

  - Change project name in GitHub template files
  - Add `Are you willing to submit a PR?` option in bug report template
  - Change some description and example in PR template
  - Change the condition in workflow from `paths-ignore` to `paths`

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not documented)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -2,28 +2,30 @@
 Thanks for sending a pull request!
 
 Here are some tips for you:
-  1. If this is your first time, please read our contributor guidelines: https://arctic.netease.com/ch/contribute/
-  2. If the PR is related to an issue in https://github.com/NetEase/arctic/issues, add '[ARCTIC-XXXX]' in your PR title, e.g., '[ARCTIC-XXXX] Your PR title ...'.
-  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][ARCTIC #XXXX] Your PR title ...'.
+  1. If this is your first time, please read our contributor guidelines: https://amoro.netease.com/how-to-contribute/
+  2. If the PR is related to an issue in https://github.com/NetEase/amoro/issues, add '[AMORO-XXXX]' in your PR title, e.g., '[AMORO-XXXX] Your PR title ...'.
+  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][ARCTIC-XXXX] Your PR title ...'.
```
Comment (zhongqishang) [inline .github/PULL_REQUEST_TEMPLATE:7]: This place `ARCTIC` is not modified.

Comment (zhoujinsong) [reply .github/PULL_REQUEST_TEMPLATE:7]: Quite right, I will change it too.

**Comment by zhoujinsong** [comment]
As the condition `paths-ignore` in workflow is not working, I would like change it back to `paths`.

**Review by baiyangtx — APPROVED** [review]
LGTM

**Pull request merged**